internal/weldr/api.go

```diff
@@ -385,13 +385,21 @@ func (api *API) openImageFile(composeId uuid.UUID, compose store.Compose) (io.Re
 	return reader, size, nil
 }
 
-// checkImageTypeDenylist checks the given ImageType name against the ImageType Denylist
-// provided to the API from configuration. If the given ImageType is not allowed the method
-// returns an `error`. Otherwise `nil` is returned.
-func (api *API) checkImageTypeDenylist(imageType string) error {
-	for _, deniedImgType := range api.imageTypeDenylist {
-		if imageType == deniedImgType {
-			return fmt.Errorf("image type denied by configuration: %q", imageType)
+// checkImageTypeDenylist checks the given ImageType and Distro names against
+//the distro-specific ImageType Denylist provided to the API from configuration.
+// If the given ImageType is not allowed the method returns an `error`.
+// Otherwise `nil` is returned.
+func (api *API) checkImageTypeDenylist(distroName, imageType string) error {
+	anyImageType := "*"
```

Comment:
I'd like to see this use actual globbing for the match. I can imagine uses where you'd want to exclude a image type from all RHEL 8 with 'rhel-8*' or all Fedora, etc. Looks like we're already using a glob library in rpmmd, `github.com/gobwas/glob` so that wouldn't add any new dependencies.

Reply:
I think it is now obvious enough, that I didn't want to get into the "globbing business" 😄 . I'll have a look at the package that we already use in rpmmd...

Reply:
I can understand not wanting to complicate things, but I think it will be a common thing to want to exclude it from a complete set of distros without needing to add new entries for each supported release.

Reply:
This is done.